portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -13,6 +13,6 @@ public class HelloWorldServlet extends HttpServlet {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Optional: Remove this servlet. 

Appears unused.

Reply:
Indeed unused; kept file but commented at top, may bring back later.